src/libraries/System.Windows.Extensions/src/System.Windows.Extensions.csproj

```diff
@@ -40,23 +46,16 @@
              Link="Common\Interop\Windows\Interop.Libraries.cs" />
     <Compile Include="$(CommonPath)Microsoft\Win32\SafeHandles\SafeHandleCache.cs"
              Link="Common\Microsoft\Win32\SafeHandles\SafeHandleCache.cs" />
-    <Compile Include="$(CommonPath)System\Text\ValueStringBuilder.cs"
-             Link="Common\System\Text\ValueStringBuilder.cs" />
     <Compile Include="System\Security\Cryptography\X509Certificates\X509Certificate2UI.cs" />
     <Compile Include="System\Security\Cryptography\X509Certificates\SafeX509Handles.cs" />
     <Compile Include="System\Security\Cryptography\X509Certificates\X509Utils.cs" />
-    <Compile Include="System\Drawing\FontConverter.cs" />
-    <Compile Include="System\Drawing\IconConverter.cs" />
-    <Compile Include="System\Drawing\ImageConverter.cs" />
-    <Compile Include="System\Drawing\ImageFormatConverter.cs" />
-    <Compile Include="System\Drawing\Printing\MarginsConverter.cs" />
     <Compile Include="System\Media\SoundPlayer.cs" />
     <Compile Include="System\Media\SystemSound.cs" />
     <Compile Include="System\Media\SystemSounds.cs" />
     <Compile Include="System\Xaml\Permissions\XamlAccessLevel.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
-    <ProjectReference Include="$(LibrariesProjectRoot)System.Drawing.Common\src\System.Drawing.Common.csproj" />
+  <ItemGroup>
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Drawing.Common\ref\System.Drawing.Common.csproj" />
```

Comment:
When do we decide to P2P to the `ref` and when do we use the `src`?

Reply:
Right now we just need the reference to the converters we're typeforwarding we no longer need anything from the implementation.